ironfish/src/rpc/routes/wallet/getTransactions.ts

```diff
@@ -99,7 +103,17 @@ router.register<typeof GetAccountTransactionsRequestSchema, GetAccountTransactio
     let count = 0
     let offset = 0
 
-    for await (const transaction of account.getTransactionsByTime()) {
+    if (request.data.sequence !== undefined && request.data.sequence < GENESIS_BLOCK_SEQUENCE) {
+      throw new ValidationError('Sequence must be a positive number')
+    }
+
+    let streamFn = () => account.getTransactionsByTime()
+    if (request.data.sequence) {
+      const sequence = request.data.sequence
```

Comment:
since you're using a closer you can just pass request into the closure without creating a new variable